packages/components/organisms/f-takeawaypay-activation/src/components/ActivationFailed.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div>
+        <bag-celebrate-bg-icon :class="$style['c-takeawaypayActivation-icon']" />
+
+        <h1
+            :class="$style['c-takeawaypayActivation-title']">
+            {{ $t('messages.titleActivationFailed') }}
+        </h1>
+
+        <p>
+            {{ $t('messages.descriptionActivationFailed') }}
+        </p>
+    </div>
+</template>
+
+<script>
+import { BagCelebrateBgIcon } from '@justeat/f-vue-icons';
+
+export default {
+
+    components: {
+        BagCelebrateBgIcon
+    }
+};
+</script>
+
+<style lang="scss" module>
+
+.c-takeawaypayActivation-icon {
+    transform: translate(10%);
+    width: 125px;
+}
+
+.c-takeawaypayActivation-title {
+    @include font-size(body-l);
+    font-weight: $font-weight-bold;
```

Comment:
Sorry for the late comment - is this not bold already? The heading default should be bold, so just wondering if that's the case or if it's getting overwritten somewhere else.

Reply:
@ashleynolan Yeah, as it is h1 now, this can be removed 👍